First Last Rules.md

```diff
@@ -0,0 +1,193 @@
+1. First Last is not allowed to point a gun at a random crew member above a contenent shaped like the State of Ohio.
+1. First Last is not allowed to "ball on" an Archon.
+1. First Last is not allowed to divulge the process of enriching uranium to the Wanderers.
+1. First Last is not allowed to eat Spanish cuisine.
+1. First Last is not allowed to export Spanish cuisine to any alien species.
+1. First Last is not allowed to own a firearm.
+1. First Last is not allowed to pay for food.
+1. First Last is not allowed to fly a surveillance drone.
+1. First Last is not allowed to name any ship after any female character of any science fiction movie franchise created between the years 1970 and 1980.
+1. First Last is not allowed to "kill all humans to give Katya a common enemy to strengthen her reign".
+1. First Last is not allowed to file a missing persons report for Michael Zahniser.
+1. First Last is not allowed to ask an Archon to "send him to the backrooms".
+1. First Last is not allowed to introduce himself as Last First.
+1. First Last is not allowed to refer to himself as anything other than "chaotic evil".
+1. First Last is not allowed to run for Republic Parliament.
+1. First Last is not allowed to "assist" Katya Reynolds in "becoming the Empress of Humanity".
+1. First Last is not allowed to install an RBMK reactor on his flagship.
+1. First Last is not allowed to refer to any Southern ship designers as "Nitrogenpocket", "Carbondioxidepocket", "Carbonmonoxidepocket", or "Watervaporpocket".
+1. First Last is not allowed to recreate the Chernobyl incident on his flagship.
+1. First Last is not allowed to hack into the Korath Automaton network and force them to play "BABY SHARK EARRAPE (10 HOUR PERFECT LOOP)".
+1. First Last is not allowed to hack into the Korath Automaton network.
+1. First Last is not allowed to form the "Paid Worlds", "Discount Worlds", "Expensive Worlds", or "Cheap Worlds".
+1. First Last is not allowed to refer to any Southern ship designers as "Methanepocket".
+1. First Last is not allowed to rename the Free Worlds government "the USSR".
+1. First Last is not allowed to argue over whether Katya Reynolds' first name is Katya or Katerina.
+1. First Last is not allowed to express his political beliefs.
+1. First Last is not allowed to express his religious beliefs.
+1. First Last is not allowed to express anything he believes in in any capacity.
+1. First Last is not allowed to refer to Katya as "Hurricane Katyna".
+1. First Last is not allowed to tribute non-human worlds.
+1. First Last is not allowed to tribute monks.
+1. First Last is not allowed to refer to any Southern ship designers using any name involving a gaseous chemical and the word "pocket".
+1. First Last is not allowed to use computers, ships, or any motorized vehicle on Harmony, Girtab.
+1. First Last is not allowed to own a Marauder Fury.
+1. First Last is not allowed to seduce an AI to do his bidding.
```

Comment:
Original rule used gender-neutral pronouns.

Reply:
The majority of the rules (as well as the general vernacular within the community) used masculine pronouns for First Last, so I normalized on that.

Reply:
Alright, but all of my rules used gender-neutral pronouns.

I'll let it slide this time, but please next time keep rules as they are, preferably change the capitalization of some letters, and only reword if necessary.

Additionally, maintain the English variant of the submitted rule, and if it uses gender neutral pronouns for First Last, keep it that way.